3-things-you-must-know-about-midnight-hacks

```diff
@@ -0,0 +1,45 @@
+
+Phillip Kinney, sophomore computer science major, was the Event Manager. When recounting the event he said, “The appeal to an overnight event like Midnight Hacks is to be able to collaborate on projects late at night and to do Smash Brothers ‘til the sun comes up.”
+
+</br>
+</br>
+##2. It’s a Party With You and 30+ of Your Closest Friends##
+
+PH is an inclusive community. New people are always welcome. Things like the Smash Brothers tournament are oriented towards students who want to meet others in a low stress atmosphere.
+
+Imanuel Sonubi, freshman computer science major, when asked about his experience at Midnight Hacks said, “I had a lot of fun just chilling with members of PH who are like family at this point. I learned about an indie game called 20xx, and also played Dungeon Souls & Smash Brothers.”
+
+![alt text](http://imgur.com/a/o3tc8)
+</br>
+</br>
+##3. It’s A Place To Get Stuff Done##
+
+Annoying roommates, city sirens, and busy coffee shops all have one thing in common- distraction. Midnight Hacks is curated to produce a positive, constructive environment. The LaunchGSU space was designed with innovation in mind (read: perfect for a fresh focus). Attendees are encouraged to bring their current class or personal project. By utilizing the diverse membership of PH, one can always find assistance if stuck or in need of a second opinion.
+
+Paul Hansa, sophomore computer science major, shared his insight, "It was a fun, relaxing night getting work done with other hackers I could get help and critique from, and the LaunchGSU was a really productive workspace to do it in."
+
+![alt text](http://imgur.com/a/TuNOA)
+
+Luis Ferrer-Labarca, junior computer science major, offered his perspective as a seasoned PantherHackers member on the event, "I've always liked the idea of hacking overnight. I believe many programmers are night owls, and the way I focus when coding overnight outperforms any of my other working routines." For the event as a whole, Ferrer adds," Overall, I had a great time, and I was able to connect with new members and see how motivated everyone was!  [Midnight Hacks was coordinated] by the Events Committee, so I want to thank them for their support and all the hard work they put into PantherHackers."
+
+![alt text}(http://imgur.com/a/y42qS)
+
+Hansa said, "MH was a really fantastic experience and a good break from all the objective-based hackathons I've been to. I want to thank Philip for being a fantastic host and helping the event go smoothly."
+
+Kinney concludes concisely, "This was a great event. I'm excited for the next one."
+
+###Whether you are looking to hang out or bang out your next app, Midnight Hacks is the event for you. Be on the lookout for the March Midnight Hacks. You can find out more information about this event on slack at pantherhackers.slack.com/signup###
```

Comment:
As a rule of thumb, leave a space after the last `#` at the beginning of the line.

e.g. `## Some title`